bugsnag-android-core/src/main/java/com/bugsnag/android/Stacktrace.kt

```diff
@@ -32,6 +32,14 @@ internal class Stacktrace : JsonStream.Streamable {
         this.logger = logger
     }
 
+    constructor(logger: Logger, frames: List<Stackframe>) {
```

Comment:
Reverse parameters for consistency with existing constructors.

```suggestion
    constructor(frames: List<Stackframe>, logger: Logger) {
```

Reply:
Discussed in-person that this isn't possible due to type-erasure causing a constructor signature clash. As these constructors are internal and not part of our public interface, no action is required here.